backend/api/views/HeatView.py

```diff
@@ -0,0 +1,131 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from api.models import MeetEvent, Heat
+from api.serializers.HeatDisplaySerializer import HeatSerializer, LaneSerializer
+from drf_spectacular.utils import extend_schema
+from django.db import transaction
+from django.db.models import Max
+from datetime import timedelta
+
+
+@extend_schema(tags=['Heat'])
+class HeatView(APIView):
+    def get(self, request, event_id, heat_num):
+        #Get event instance
+        try:
+            event_instance = MeetEvent.objects.get(id=event_id)
+        except MeetEvent.DoesNotExist:
+            return Response({'error': 'Event not found'}, status=status.HTTP_404_NOT_FOUND)
+        
+        #Get num_lanes
+        try:
+            swim_meet_instance = event_instance.swim_meet
+            num_lanes = swim_meet_instance.site.num_lanes
+        except:
+            return Response({'error': 'Not able to retrieve number of lanes'}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
This response is returning 404 but the message is general. I would suggest to return a more general code or more specific message for 404.

Reply:
I changed the message to 'Number of lanes not found for the swim meet site.'
What do you think?